### PR TITLE
Docs issue-3444: draft example use case and infraconfig examples

### PR DESCRIPTION
**Pull request by jenmalloy**
### Type of change
 Documentation

### Description
Example use case and brokered infraconfig examples (other service config resource type examples to follow in subsequent commits)

### Checklist
- [ ] Update/write design documentation in `./documentation/design`
- [ ] Write tests and make sure they pass
- [X] Update documentation
- [ ] Check RBAC rights for Kubernetes / OpenShift roles
- [ ] Try your changes from Pod inside your Kubernetes and OpenShift cluster, not just locally
- [ ] Reference relevant issue(s) and close them after merging
- [ ] Update CHANGELOG.md

### CODE REVIEW

File: documentation/modules/ref-ability-restrict-address-space-connections.adoc [hunk 1]
```diff
@@ -0,0 +1,49 @@
+// Module included in the following assemblies:
+//
+// assembly-configuring.adoc
+
+[id='ref-ability-restrict-address-space-connections-{context}']
+= Ability to restrict address space connections
+
+Company X has the following requirement for using {ProductName}: For engineering work, the ability to restrict who can connect into the address space is required.
+
+To meet this requirement you must set a network policy in the brokered infrastructure configuration. For more information about network policies, see OpenShift documentation.
```
Comment (lulf) [inline documentation/modules/ref-ability-restrict-address-space-connections.adoc]: Is it possible to link to Kubernetes/OpenShift docs here?

File: documentation/modules/ref-auth-service-reource-examples.adoc [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+----
+apiVersion: admin.enmasse.io/v1beta1
+kind: AuthenticationService
+metadata:
+  name: engineering
+spec:
+  type: none
+----
+
+For QA work, the QA team must be able to create users for each instance where the credentials are verified. Also,  QA has a database they want to use for persisting the users. To meet this requirement, you must use the `standard` authentication service and specify a data source:
```
Comment (lulf) [inline documentation/modules/ref-auth-service-reource-examples.adoc]: Drop 'where the credentials are verified'.

File: documentation/modules/ref-example-use-case.adoc [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+* For engineering work, restricting the messaging infrastructure to support storage of no more than 1000 messages of approximately 1 KB per message, with up to 10 queues and topics is required.
++
+For QA work, restricting the messaging infrastructure to support storage of no more than 10,000 messages of approximately 100 KB, with up to 50 queues and topics is required.
+
+* For engineering work, the ability to restrict who can connect into the address space is required.
+
+* For engineering work, the engineering team does not need to create distinct users that need to be individually authenticated.
++
+For QA work, the QA team must be able to create users for each instance where the credentials are verified.
```
Comment (lulf) [inline documentation/modules/ref-example-use-case.adoc]: "The QA team must be able to create users for each address space"  and drop "where the credentials are verified".

File: documentation/modules/ref-ability-restrict-address-space-connections.adoc (hunk 1 above)
Comment (jenmalloy) [reply documentation/modules/ref-ability-restrict-address-space-connections.adoc]: My bad; will push a fix

**Comment by jenmalloy** [comment]
@lulf, Could  you please take another look? Added content and addressed your previous review comments

**Pull request merged**